packages/react/src/formElements/MdAutocomplete.tsx

```diff
@@ -54,7 +54,7 @@ const MdAutocomplete: React.FunctionComponent<MdAutocompleteProps> = ({
   const [results, setResults] = useState<MdAutocompleteOptionProps[]>([]);
 
   const uuid = id || uuidv4();
-  const inputId = id && id !== '' ? id : uuidv4();
+  const inputId = uuidv4() as string;
```

Comment:
Går ikke dette ann her?
 `const inputId = uuid as string`
 Ser for meg at det kan være lurt at input-elementet får id'n fra props hvis den er satt 🤔